src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -9,16 +9,21 @@
         <i class="fas fa-cog" aria-hidden="true"></i>
     {{/button}}
   {{/page-header-tools-group}}
-  {{#> page-header-tools-group}}
-    {{#> dropdown id=(concat page--id '-dropdown-kebab-right-aligned-1') dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-mobile pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+  {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-user-dropdown"}}
```

Comment:
did you mean to add `pf-m-user-dropdown` in this PR? If so you can use that to target the avatar and potentially toggle underline styles if you wan to make all of those changes part of this PR.